ocs_ci/deployment/hosted_cluster.py

```diff
@@ -1814,6 +1824,25 @@ def wait_console_plugin_pod_running(self):
                 )
         return False
 
+    @catch_exceptions((CommandFailed, TimeoutExpiredError))
+    def catalog_source_on_management(self):
+        """
+        Check if imageStream catalogs exists on Managment cluster.
+        In this case catalogSource on Guest cluster can not be overwritten.
+
+        Returns:
+            bool: True if catalogSource exists on Management cluster, False otherwise
```

Comment:
I see that the _tox: docs_ is not failing on this, but the list in markdown should end with one empty line here.